prediction_market_agent/agents/autogen_general_agent/prompts.py

```diff
@@ -1,5 +1,4 @@
-INFLUENCER_PROMPT = """
-        You are an AI agent that places bets on future events. 
+INFLUENCER_PROMPT = """You are an AI agent that places bets on future events. 
         You can find below a list of recent BETS that you have placed.
```

Comment:
Is the rest of the whitespace intended to keep?

Btw you can use openai's online tokenizer to test this stuff:

![Screenshot by Dropbox Capture](https://github.com/gnosis/prediction-market-agent/assets/15619339/29aff815-51ae-450d-8d64-f2f3c1fa8b89)

Reply:
Good to know, the empty space is counted as well. I will optimize that.